docs/secondary-network.md

```diff
@@ -58,12 +57,8 @@ metadata:
   name: antrea-config
   namespace: kube-system
 data:
-  antrea-controller.conf: |
```

Comment:
We shouldn't remove it directly, can you refer to this commit to refine the description?
https://github.com/antrea-io/antrea/commit/8cbd5ba7e538c0ed2f74ad01ad7352c557cca38c

Reply:
updated.